code/modules/mob/living/basic/vermin/stickman.dm

```diff
@@ -0,0 +1,85 @@
+/mob/living/basic/stickman
+	name = "Angry Stick Man"
+	desc = "A being from a realm with only 2 dimensions. At least it's trying to stay faced towards you."
+	icon_state = "stickman"
+	icon_living = "stickman"
+	icon_dead = "stickman_dead"
+	mob_biotypes = MOB_HUMANOID
+	gender = MALE
+	health = 100
+	maxHealth = 100
+	speed = 0
+	attack_verb_continuous = "punches"
+	attack_verb_simple = "punch"
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	attack_sound = 'sound/weapons/punch1.ogg'
+	combat_mode = TRUE
```

Comment:
i don't think this is needed (?) can u test?

Reply:
isnt this what prevents you from swapping places

Reply:
oh true lol